bookwyrm/templates/components/modal.html

```diff
@@ -18,7 +23,7 @@ <h2 class="modal-card-title">
         </footer>
         {% block modal-form-close %}{% endblock %}
     </div>
-    <label class="modal-close is-large" for="{{ controls_text }}-{{ controls_uid }}" aria-label="close"></label>
+    <button class="modal-close is-large" aria-label="close"></button>
```

Comment:
this line should actually be deleted completely! that's my bad, I forgot to remove it when I refactored this template. The line below it adds the same close button but is actually functional